xmodule/library_content_block.py

```diff
@@ -451,7 +451,12 @@ def author_view(self, context):
         context['is_loading'] = is_updating
 
         # The following JS is used to make the "Update now" button work on the unit page and the container view:
-        fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/library_content_edit.js'))
+        if 'library' in root_xblock.category:
+            if root_xblock.source_library_id and len(root_xblock.children) > 0:
```

Comment:
dumb question: should the view button exist if the library is selected, but has no content?
You're allowed to say no, and that we shouldn't care.

Reply:
I don't think that we need to card. I think for consistency we should keep the view button any time there is not a configuration error.